packages/@lwc/integration-not-karma/helpers/matchers.mjs

```diff
@@ -0,0 +1,31 @@
+// FIXME: copy implementations from karma tests
+
+const customMatchers = [
+    // LWC custom
+    'toLogError',
+    'toLogErrorDev',
+    'toThrowErrorDev',
+    'toLogWarningDev',
+    'toThrowCallbackReactionError',
+    'toThrowCallbackReactionErrorDev',
+    'toThrowCallbackReactionErrorEvenInSyntheticLifecycleMode',
+    // jasmine compat
+    'toHaveSize',
+    'toBeFalse',
+    'toBeTrue',
+];
+export const registerCustomMatchers = (chai, utils) => {
+    for (const matcher of customMatchers) {
+        utils.addMethod(chai.Assertion.prototype, matcher, function () {
+            // FIXME: implement for realsies
+            const fn = utils.flag(this, 'object');
+            if (typeof fn === 'function') {
+                try {
+                    fn();
+                } catch (_) {
+                    //
+                }
+            }
+        });
+    }
+};
```

Comment:
Just making sure I understand: this is a placeholder, where actual implementations of the custom matchers will be implemented in future PRs? If so, 👍